src/components/Toolbar/ActionMenu.js

```diff
@@ -24,15 +25,31 @@ import React, {Component} from 'react';
  * */
 
 export default class ActionMenu extends Component {
+  containerRef = React.createRef();
+
   constructor() {
     super();
     this.state = {
       active: false,
     };
   }
 
+  handleMouseDown = (e) => {
+    const refNode = findDOMNode(this.containerRef.current);
```

Comment:
What happens if you don't use `findDOMNode` here?

Reply:
it'll set it to close and then handleClick on the dropdown menu which triggers after will open it (so clicking on the button no longer closes the menu). couldn't think of a nice way to handle that case other than checking the node

Reply:
wow that's rough. does `mouseMove={e=>e.stopPropagation()}` do anything?

Reply:
to handle this case? it shouldnt, the mouse doesnt actually move between mousedown and mouseup. also prob not good to randomly stop propagation of global mouse move event :P 

im also happy to close this in favor of using a library (material-ui) that already supports outside click to close

Reply:
i meant attaching that to the dropdown menu itself, so that the event doesn't bubble up to window